ADOPTERS.md

```diff
@@ -0,0 +1,15 @@
+ADOPTERS
+========
+
+PUBLIC REFERENCES
+-----------------
+
+This document lists the organizations that use Idpbuilder based on public information available in blog posts and videos.
+If any organization would like to get added or removed please make a pull request by following these guidelines:
+
+* Please don't include your organization's logo or other trademarked material
+* Add a reference (link to a public blog post, video, slides, etc) mentioning that Idpbuilder is used
```

Comment:
```suggestion
* Add a short description on how your organization is benefiting from using idpbuilder,  is encourage to also include link to a public (blog post, video, slides, git repo, etc.) that shows or mentions how Idpbuilder is used
```

There might organizations using it internally, and don't have a public url reference, and we want to be able to capture this adopters in this document.